### PR TITLE
Allow default status, priority, severity and type in issues and tasks

### PR DESCRIPTION
**Pull request by jespino**

### CODE REVIEW

File: tests/integration/test_userstories.py [hunk 1]
```diff
@@ -45,6 +45,23 @@ def test_update_userstories_order_in_bulk():
...
+    project.save()
+
+    f.MembershipFactory.create(project=project, user=user, is_owner=True)
+    url = reverse("userstories-list")
+
+    data = {"subject": "Test user story", "project": project.id}
+    client.login(user)
+    response = client.json.post(url, json.dumps(data))
+    assert response.status_code == 201
+    assert json.loads(response.content)['status'] == project.default_us_status.id
```
Comment (bameda) [inline tests/integration/test_userstories.py]:
```
 assert response.data['status'] == project.default_us_status.id
```

File: tests/integration/test_tasks.py [hunk 2]
```diff
@@ -36,6 +36,23 @@ def test_create_tasks_in_bulk(db):
...
+    project.save()
+
+    f.MembershipFactory.create(project=project, user=user, is_owner=True)
+    url = reverse("tasks-list")
+
+    data = {"subject": "Test user story", "project": project.id}
+    client.login(user)
+    response = client.json.post(url, json.dumps(data))
+    assert response.status_code == 201
+    assert json.loads(response.content)['status'] == project.default_task_status.id
```
Comment (bameda) [inline tests/integration/test_tasks.py]: assert response.data['status'] == project.default_task_status.id

File: tests/integration/test_issues.py [hunk 3]
```diff
@@ -46,6 +46,32 @@ def test_update_issues_order_in_bulk():
...
+    data = {"subject": "Test user story", "project": project.id}
+    client.login(user)
+    response = client.json.post(url, json.dumps(data))
+    print(response.content)
+    assert response.status_code == 201
+    assert json.loads(response.content)['status'] == project.default_issue_status.id
+    assert json.loads(response.content)['severity'] == project.default_severity.id
+    assert json.loads(response.content)['priority'] == project.default_priority.id
+    assert json.loads(response.content)['type'] == project.default_issue_type.id
+
```
Comment (bameda) [inline tests/integration/test_issues.py]:
```
    assert response.data['status'] == project.default_issue_status.id
    assert response.data['severity'] == project.default_severity.id
    assert response.data['priority'] == project.default_priority.id
    assert response.data['type'] == project.default_issue_type.id
```

**Comment by jespino** [comment]
Thanks for the changes :)

File: tests/integration/test_issues.py [hunk 4]
```diff
@@ -46,6 +46,32 @@ def test_update_issues_order_in_bulk():
...
+    project.default_severity = severity
+    project.default_issue_type = type
+    project.save()
+    f.MembershipFactory.create(project=project, user=user, is_owner=True)
+    url = reverse("issues-list")
+
+    data = {"subject": "Test user story", "project": project.id}
+    client.login(user)
+    response = client.json.post(url, json.dumps(data))
+    print(response.content)
```
Comment (superalex) [inline tests/integration/test_issues.py]: Shouldn't this print be removed?

**Comment by superalex** [comment]
Great! :+1:

**Pull request merged**